eip-7534.md

```diff
@@ -0,0 +1,221 @@
+---
+eip: 7534
+title: TCT
+description: TCT combines the benefits of runtime checking and symbolic proof. The unique design of TCT ensures that the theorems are provable and checkable in an efficient manner.
+author: Shuo Chen (@cs0317), Nikolaj Bjørner (@NikolajBjorner), Tzu-Han Hsu (@tzuhancs), Ashley Chen (@ash-jyc), Nanqing Luo (@Billy1900)
+discussions-to: https://discord.gg/WJKNcsudR9
```

Comment:
Please create a "discussions-to" thread at [FEM](https://ethereum-magicians.org/)

Reply:
the link is here: https://ethereum-magicians.org/t/eip-7534-transactioncarrying-theorem-tct-proposal/17125. Should I put it in the readme?

Reply:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7534-transactioncarrying-theorem-tct-proposal/17125
```

Reply:
The Discord link isn't allowed for discussion, replace it with FEM.